### PR TITLE
Langton Ant implemented in javascript

### PR DESCRIPTION
**Pull request by mitch3b**
**Fixes issue:** #1845

**Changes:**
Langton Ant implemented in javascript

### CODE REVIEW

File: code/cellular_automaton/langtons_ant/LangtonAnt.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
```
Comment (arnavb) [inline code/cellular_automaton/langtons_ant/LangtonAnt.html]: You may want to consider fixing the HTML indentation for clarity.

File: code/cellular_automaton/langtons_ant/LangtonAnt.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+</script>
+</head>
+
+<body>
+
+<h2>Langton Ant</h2>
+
+<button type="button" onclick="nextMove()">Step</button><br><br>
+
+<div style="position: relative;">
```
Comment (arnavb) [inline code/cellular_automaton/langtons_ant/LangtonAnt.html]: Move this styling to the style tag.

**Comment by mitch3b** [comment]
Yep and yep

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**